src/pages/Members/Members.scss

```diff
@@ -0,0 +1,47 @@
+@use '../../styles/variables' as var;
+@use '../../styles/mixins' as mix;
+
+.members-section {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 4rem 6rem 8rem 6rem;
+
+  .members-section-left {
+    max-width: 50%;
+    padding-left: 4rem;
+  }
+  .members-section-right {
+    width: 50%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+  .members-illustration {
+    width: 40%;
+  }
+
+  .members-description {
+    max-width: 100%;
+    font-weight: var.$semibold;
+    padding-top: 3rem;
+  }
+}
+
+.heading2-underlined {
+  @include mix.heading2;
+  position: relative;
+  display: inline-block;
```

Comment:
It is suspiciously a lot of inline-blocks. We have 5 of them in our 100k LOC project

<img width="319" alt="CleanShot 2024-11-05 at 21 27 33" src="https://github.com/user-attachments/assets/468169b6-234d-40e5-9bd7-dd0729bbb2b9">

you have 4 of them here. It is completely fine, if it is needed. I am just not used to seeing it :D